OIPA/iati_organisation/parser/organisation_2_03.py

```diff
@@ -215,6 +215,48 @@ def iati_organisations__iati_organisation(self, element):
 
             return element
 
```

Comment:
I think the pull request should not include this method (because the pull request is about name__narrative and not name), also, (I think) the logic here is not 100% correct (see: https://github.com/zimmerman-zimmerman/OIPA/pull/1040#pullrequestreview-202050597)